sql-cli/config/astro-cli.json

```diff
@@ -1,3 +1,9 @@
 {
-  "baseDockerImage": "quay.io/astronomer/astro-runtime:6.0.4-base"
+  "baseDockerImage": "quay.io/astronomer/astro-runtime:6.0.4-base",
+  "compatibility": {
+    "0.3.0a1": {
+      "astro-runtime": "8.0.0",
+      "astro-sdk-python": "1.3.0"
+    }
+  }
```

Comment:
@pankajkoti how hard would it be for us to change the JSON to:
```
{
    "compatibility": {
        "0.3": {
            "astroSDKPythonMaxVersion": "",
            "astroSDKPythonMinVersion": "1.3.1",
            "astroRuntimeMinVersion": "6.1.0",
            "astroRuntimeMaxVersion": "7.1.0"
        }
}
```
https://github.com/astronomer/astro-sdk/issues/1473

Sorry, I updated the ticket last week after you started working on it, when I was reviewing the tickets